database.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+# module CreateDatabase
+module Database
+  PATH_FILE = 'seeds.yaml'.to_s
+
+  def save_data
+    data = {
+      authors: @authors,
+      books: @books,
+      readers: @readers,
+      orders: @orders
+    }
+
+    File.write(PATH_FILE, data.to_yaml)
+  end
+
+  def load_data
+    if File.exist?('seeds.yaml')
+      yaml_file = File.read(PATH_FILE)
```

Comment:
i think, it will be better to move code from if statement to (for example) `load_library_from_seeds` method and do like:
`
data = File.exists?(PATH_FILE) ? load_library_from_seeds : generate_data
add_entity_to_library(data)
`

Reply:
Thanks for the good advice

Reply:
Thanks for the good advice